mnefun/_paths.py

```diff
@@ -242,3 +242,7 @@ def _prebad(p, subj):
 def _is_dir(d):
     """Safely check for a directory (allowing symlinks)"""
     return op.isdir(op.abspath(d))
+
+
+def _get_config_file():
+    return op.expanduser(op.join('~', '.mnefun', 'mnefun.json'))
```

Comment:
A user-set environmental variable with the name of JSON file is another way to go, but I see that the ".mnefun" directory is analogous to the ".mne/" directory already specified for MNE-Python.

Reply:
We can do it someday if we need to but for now I'd assume YAGNI